tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "outDir": "./lib",
+    "outDir": "./lib/cjs",
```

Comment:
This change will break customers that import specific files from `@contentful/node-apps-toolkit` as the file paths change. This is not a blocker, but depending on how liberal we are with major version bumps it might be worthwhile to bump the package to 3.0.0 when merging this change.

Reply:
The package is versioned plus importing internal files is not part of the public api
So I think we are fine with that change.